daml-foundations/daml-tools/daml-extension/syntaxes/daml12.tmLanguage-update.xml

```diff
@@ -463,7 +456,7 @@
                   )
                   (?:\s*,\s*\g&lt;fn&gt;)?
                 )
-                \s*(::|∷)</string>
+                \s*(:|:)</string>
```

Comment:
Isn't `:|:` equivalent to just `:`?

Reply:
Ah yes, thanks for pointing out, will fix it